components/folder/LinkAdd.styled.ts

```diff
@@ -0,0 +1,32 @@
+import styled from "styled-components";
+
+export const LinkAddWrap = styled.form`
```

Comment:
태그의 역할을 매칭해서 컴포넌트이름을 지으면 좋아요

```suggestion
export const LinkAddForm = styled.form`
```